src/State/Provider/ContentNegotiationProvider.php

```diff
@@ -121,4 +121,19 @@ private function getInputFormat(HttpOperation $operation, Request $request): ?st
 
         return null;
     }
+
+    private function getOpenApiRequestBodyFormats(HttpOperation $operation): array
```

Comment:
we don't want a link between State and OpenApi, but why can't you add the `multipart` option to this operation only?